code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -820,6 +820,7 @@
 	var/locked = TRUE
 	var/control_area = null //can be area name, path or nothing.
 	var/ailock = 0 // AI cannot use this
+	var/shoot_cyborgs = 0

```

Comment:
FALSE

Reply:
Its an option now, heads can turn it on for AI uploads and shit, Im confused on what you want me to do, turn the option on from the start? Allow silicons to have the option? Probably just the AI honestly. Ill do that.

Reply:
`FALSE` is a define used for bools, don't use `1` or `0`; use `TRUE` or `FALSE`

Reply:
Ahhh okay.

Reply:
I will admit Im not the one who set these booleans to 1 or 0.

Reply:
T/F didn't exist whenthese were added